### PR TITLE
Fix ICO funds raised

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: lib/sanbase/model/ico.ex [hunk 1]
```diff
@@ -116,16 +116,27 @@ defmodule Sanbase.Model.Ico do
...
+          price =
+            Sanbase.Price.Utils.fetch_last_price_before(
+              slug,
+              target_currency,
+              datetime
+            )
+
+          case price != nil and ic.amount != nil do
+            true ->
+              {price, ic.amount}
```
Comment (tspenov) [inline lib/sanbase/model/ico.ex]: this tuple is not needed

Comment (IvanIvanoff) [reply lib/sanbase/model/ico.ex]: yeah, inspect leftover

File: lib/sanbase/model/project/funds_raised.ex [hunk 2]
```diff
@@ -24,6 +24,7 @@ defmodule Sanbase.Model.Project.FundsRaised do
     end
   end
 
+  @spec initial_ico(Sanbase.Model.Project.t()) :: any
```
Comment (tspenov) [inline lib/sanbase/model/project/funds_raised.ex]: `float | nil` instead of any ?

Comment (IvanIvanoff) [reply lib/sanbase/model/project/funds_raised.ex]: it's date, not float

File: lib/sanbase/prices/price_sql_query.ex [hunk 3]
```diff
@@ -300,7 +300,7 @@ defmodule Sanbase.Price.SqlQuery do
     """
 
     # Put an artificial lower boundary otherwise the query is too slow
-    from = Timex.shift(datetime, days: -180) |> DateTime.to_unix()
+    from = Timex.shift(datetime, days: -30) |> DateTime.to_unix()
```
Comment (tspenov) [inline lib/sanbase/prices/price_sql_query.ex]: extract as named module attribute

File: lib/sanbase/model/project/funds_raised.ex (hunk 2 above)
Comment (tspenov) [reply lib/sanbase/model/project/funds_raised.ex]: ups,  I was fooled by the diff - I thought it is the result of this 
```
      [] ->
      amounts -> Enum.reduce(amounts, 0, &Kernel.+/2)	        nil

      amounts ->
        Enum.reduce(amounts, 0, &Kernel.+/2)
```

File: lib/sanbase/prices/price_sql_query.ex (hunk 3 above)
Comment (IvanIvanoff) [reply lib/sanbase/prices/price_sql_query.ex]: In case this is used only here and is an implementation detail i think the comment is enough

**Pull request merged**